data/schema/schema_v1.json

```diff
@@ -599,7 +614,8 @@
                               }
                             },
                             "calculated": {
-                              "type": "boolean"
+                              "type": "boolean",
+                              "description": "Value calculated from answers."
```

Comment:
This isn't a calculated value, it's a boolean that indicates if this answer should be treated as the result of a calculation.

Reply:
Indeed. New text

"Shows that answer is to be treated as result of calculation or not."